argusclient/model.py

```diff
@@ -450,6 +450,12 @@ def __init__(self, *args, **kwargs):
     def from_json(self, jsonObj):
         if not jsonObj or not isinstance(jsonObj, dict):
             return jsonObj
+
+        # temp workaround to argus bug
```

Comment:
This was added as a workaround to an argus issue, where 2 different endpoints return 'notifier' and 'notifierName' in the notifications. This caused issues in argusclient where this decoder method could not recognize the notification as a Notification object, since it didn't have the 'notifierName' field (instead, it had the 'notifier' field).

Reply:
Couple of thoughts:
- Is this a temp workaround because a fix is coming up on the service side? If so, how to track it so that one would know if this workaround can be removed. If there is a bug that can be watched, it can be linked from the above comment.
- Even if it is a temp workaround, we could put more details into the comment, basically the above comment or a condensed version of it.
- Perhaps we should make a permanent provision to have aliases for fields and not make this a temp workaround? In the `id_fields`, we currently have strings, but we could also support list of strings and then adjust the logic in `from_dict`.

Reply:
I think point 3 sounds good, but I dont think itll work, because the __init__ method expects a variable called 'notifierName' and does validation on it.

Added more details to the comment, linked the bug item, and made it as concise as I could

Reply:
Yes, you are right about the impact on `_init_` too, but the below should work.

```
    def __init__(self, name, notifierName=None, metricsToAnnotate=None, **kwargs):
        notifierName = notifierName or kwargs.get('notifier')
        assert notifierName in Notification.VALID_NOTIFIERS, "notifierName is not valid: %s" % notifierName
        super(Notification, self).__init__(name=name, notifierName=notifierName, metricsToAnnotate=metricsToAnnotate or [], **kwargs)
````

The above will ignore `notifier` if both exist, which is fine (we can just document the behavior), but we can easily add another check if needed. What do you think?
What do you think?

Reply:
@haridsv Ah nice! it works. Had to make a few minor changes to get it to work btw. Please check again